hassio/src/dialogs/registries/dialog-hassio-registries.ts

```diff
@@ -116,13 +125,18 @@ class HassioRegistriesDialog extends LitElement {
                   Add new registry
                 </mwc-button> `}
         </div>
-        <mwc-button slot="primaryAction" @click=${this.closeDialog}>
-          Close
-        </mwc-button>
       </ha-dialog>
     `;
   }
 
+  private _inputChanged() {
+    this._entriesAreFilled = Boolean(
+      this._newRegistry?.value &&
+        this._newUsername?.value &&
+        this._newPassword?.value
+    );
+  }
```

Comment:
Save the values of the 3 inputs in private properties instead of querying all 3 on every key